assembl/static/js/app/views/tokenVoteSession.js

```diff
@@ -1303,8 +1321,8 @@ var TokenResultView = Marionette.LayoutView.extend({
     var settings = this.model.get("settings") || {},
         items = "items" in settings ? settings.items : [],
         questionItem = items.length ? items[0] : null,
-        questionTitle = "question_title" in questionItem ? questionItem.question_title : "",
-        questionDescription = "question_description" in questionItem ? questionItem.question_description : "",
+        questionTitle = this.questionTitle ? this.questionTitle : "question_title" in questionItem ? questionItem.question_title : "",
```

Comment:
This will show the original langstring value if the user has no langprefs?

Reply:
Yes and also until the langprefs have been received, and also if there is any issue with the received langprefs